Applications/Spire/Include/Spire/KeyBindings/CancelKeyBindingsTableModel.hpp

```diff
@@ -9,28 +9,31 @@ namespace Spire {
   class CancelKeyBindingsTableModel : public QAbstractTableModel {
     public:
 
+      using Action = KeyBindings::CancelAction;
+
+      using Binding = KeyBindings::CancelActionBinding;
+
       //! Constructs a CancelKeyBindingsTableModel with initial key
       //! bindings.
       /*
         \param bindings The initial key bindings to display.
         \param parent The parent object.
       */
       explicit CancelKeyBindingsTableModel(
-        const std::vector<KeyBindings::CancelActionBinding>& bindings,
-        QObject* parent = nullptr);
+        const std::vector<Binding>& bindings, QObject* parent = nullptr);
 
       //! Updates the key bindings to display.
       /*
         \param bindings The updated key bindings.
       */
       void set_key_bindings(
-        const std::vector<KeyBindings::CancelActionBinding>& bindings);
+        const std::vector<Binding>& bindings);
 
       //! Returns the cancel action associated with the given row.
       /*
         \param row The row to get the associated cancel action from.
       */
-      KeyBindings::CancelAction get_cancel_action(int row);
+      static Action get_cancel_action(int row);
```

Comment:
I think static methods should be declared between inner class types and non-static methods, shouldn't they? It is very unlikely that they should be placed between non-static methods anyway.

Reply:
Isn't that where it's currently declared? It's after the inner classes/structs and before the other functions.